selfdrive/manager/build.py

```diff
@@ -80,6 +80,9 @@ def build(spinner, dirty=False):
           t.wait_for_exit()
         exit(1)
     else:
+      shutil.rmtree("/tmp/scons_cache", ignore_errors=True)
+      shutil.rmtree("/data/scons_cache", ignore_errors=True)
+      os.system(f"cd {BASEDIR} && scons {j_flag} --cache-populate")
```

Comment:
This will add quite a bit of time to the build process. Sure we want to do this?

Reply:
changed it to enforce a max size instead of just keeping the last build products. new implementation is pretty fast